MANIFEST.in

```diff
@@ -1 +1,2 @@
 include README.rst
+prune workalendar/tests*
```

Comment:
Strange way do to that. I though adding tests in the package was to ship them as part of the code.
If you do not want to ship them you should probably put them outside the package.